### ISSUE TITLE
lowercase-dash rule allows for underscores

### ISSUE THREAD

**Issue by GrayedFox** [jeanlauliac/fs-lint#5]
The lowercase dash rule doesn't catch file names that have an underscore.

For example `./bin/oath2_proxy` won't trigger an error with the following rule:

```
{
  "rules": [
    {
      "files": [
        "bin/**/*"
      ],
      "naming": "lowercase-dash",
      "level": "error"
    }
  ]
}
```
I know the rule is working because if I rename the file to `./bin/oath2_proxY` fs-lint will trigger the error:

```
error: violates 'lowercase-dash' naming: bin/oauth2-proxy-2-2-0-linux-amd64-go1-8-1/oauth2_proxY
```
